### DISCUSSION TITLE
Mark casks as updated?

### DISCUSSION THREAD

**Discussion by kaelri** [Homebrew/discussions#1288] [Casks]
I've been brewing a long while now, and `brew outdated --greedy` gives me a lengthy list of auto-upgrading casks, which Homebrew has (rightly) not upgraded since I installed them. I'm wondering whether there's a way that I can simply tell Homebrew that these apps are now on the latest version, without making it actually perform a redundant upgrade.

No real practical need for this, I admit—just my own peace of mind. I realize too that it can't work for casks with a version of `latest`. Beg pardon if there's a known solution, I did search before posting!

**Comment by SMillerDev** [comment]
Not that I know of, but pinging @homebrew/cask for a definitive answer.

**Reply by kaelri** [reply]
Thanks for checking!

**Comment by vitorgalvao** [comment]
No, there’s no official way.

**Reply by kaelri** [reply]
Alas. Thanks anyway!
